src/vt_flash/newton.jl

```diff
@@ -224,27 +297,31 @@ function vt_flash_newton!(
         newton_helmhess!,
         statex;
         constrain_step=constrain_step,
-        gtol=gtol,
+        gtol=NaN,
         maxiter=maxiter,
+        convcond=convcond,
     )
     statex .= optimresult.argument
 
     return __vt_flash_two_phase_result(state, mix, nmol, volume, RT, optimresult)
 end
 
 """
-    vt_flash_newton(mix, nmol, volume, RT)
+    vt_flash_newton(mix, nmol, volume, RT, StateVariables[; chemrtol, pressrtol, maxiter])
 
 Find VT-equilibrium of `mix`, at given `nmol`, `volume` and thermal energy `RT`
-using Newton's minimization. Return [`VTFlashResult`](@ref).
+using Newton's minimization.
+
+Return [`VTFlashResult`](@ref).
 """
 function vt_flash_newton(
     mix::BrusilovskyEoSMixture,
     nmol::AbstractVector,
     volume::Real,
     RT::Real,
     StateVariables::Type{<:AbstractVTFlashState};
-    gtol::Real=1e-3/RT,
+    chemrtol::Real=1e3*eps(),
+    pressrtol::Real=1e3*eps(),
```

Comment:
As we use relative tolerances we may guarantee no more than machine precision.

Reply:
I'd propose `1024 * eps(T)` (where `T` is defined by the eltype of vectors passed to optimizer) to relax tolerances in case `Float32` is used.

Reply:
Fixed by 013107f248a970354e9130944d0b3fd3baa6b9ca